test/MUXControlsTestApp/TAEF/project.json

```diff
@@ -3,7 +3,8 @@
     "Microsoft.Net.Native.Compiler": "2.2.1",
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
     "MUXCustomBuildTasks": "1.0.38",
-    "TAEF.Redist.Wlk": "10.31.180822002"
+    "TAEF.Redist.Wlk": "10.31.180822002",
+    "Win2D.uwp": "1.22.0"
```

Comment:
I thought you updated the sample to not need Win2D?

Reply:
Yes, will fix this.